lib/src/repo.rs

```diff
@@ -119,47 +119,47 @@ pub enum RepoInitError {
 }
 
 impl ReadonlyRepo {
-    pub fn default_op_store_factory() -> impl FnOnce(&Path) -> Box<dyn OpStore> {
-        |store_path| Box::new(SimpleOpStore::init(store_path))
+    pub fn default_op_store_initializer() -> Box<OpStoreInitializer> {
+        Box::new(|store_path| Box::new(SimpleOpStore::init(store_path)))
```

Comment:
nit: doesn't have to be boxed?

`-> &'static OpStoreInitializer` works so long as the closure can be turned into function pointer.

Reply:
Ah, I didn't think of making it `&'static`. Done.